Bug 1970969: clarify failure message

cheesesashimi opened the pull request:
The failure message for "[sig-cluster-lifecycle] cluster upgrade should be fast" is ambiguous:

upgrade to registry.build01.ci.openshift.org/ci-op-h21l7wld/release@sha256:2148b1c121946ac4f186bb22b166247d3df0ad9cf3966f05dc2a6ea27bc53927 took too long: 86.33481681223333

What does 86.33481681223333 represent? Is it seconds, minutes, hours? What does "too long" mean? Without looking at the test code, one cannot tell. With that info added to the failure string, it's easier to understand:

upgrade to registry.build01.ci.openshift.org/ci-op-h21l7wld/release@sha256:2148b1c121946ac4f186bb22b166247d3df0ad9cf3966f05dc2a6ea27bc53927 took too long: 86.3 minutes, expected 75 minutes or less

I'm happy to change verbiage as needed.

vrutkovs commented:
/retest

Upgrade tests didn't seem to hit it

vrutkovs commented:
/test e2e-aws-upgrade

vrutkovs commented:
cluster-bot's [test upgrade 4.7 openshift/origin#26207 gcp](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-gcp/1402193022900244480) shows:

>upgrade to registry.build01.ci.openshift.org/ci-ln-d63f0b2/release@sha256:eb1d8b95d82f9eea19846aa019c2bc5a817e729e8ebee0b3159e5759dd92f11f took too long: 80.33 minutes, expected: 75 minutes or less

/lgtm

vrutkovs commented:
/assign soltysh
needs approval

soltysh reviewed (approved):
/approve
/retest
@cheesesashimi @vrutkovs you'll need a valid BZ to land it, though

vrutkovs commented:
I'm fine with this merging in 4.9 and being backported later

cheesesashimi commented:
Sorry about the lack of a BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1970969. I'm not sure if I opened it against the correct component or not, so please adjust if needed.

vrutkovs commented:
/bugzilla refresh

Needs rebase, as I tweaked the error message recently

cheesesashimi commented:
Fixed. Also, it looks like the non-AWS upgrade timeout is 75 minutes (https://github.com/openshift/origin/blob/master/test/e2e/upgrade/upgrade.go#L274). I've adjusted the info message from 60m to 75m.

vrutkovs commented:
/lgtm
